logger/dapr_logger.go

```diff
@@ -0,0 +1,146 @@
+package logger
+
+import (
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// daprLogger is the implemention for logrus
+type daprLogger struct {
+	// name is the name of logger that is published to log as a scope
+	name string
+	// loger is the instance of logrus logger
+	logger *logrus.Entry
+}
+
+var GetVersion = func() string {
```

Comment:
remove this global var and get version via param in the method where this is used. See comment below.